templates/main.php

```diff
@@ -4,6 +4,7 @@
  * @var \OCP\IL10N $l
  */
 \OCP\Util::addScript('oc-backbone-webdav');
+vendor_script('core', ['marked/marked.min']);
```

Comment:
As per https://github.com/nextcloud/server/pull/13474 marked is being removed from core,
so we have to ship it ourselves. I will take a look at your PR now and fix this.